### DISCUSSION TITLE
request a new release

### DISCUSSION THREAD

**Discussion by chenrui333** [helix-editor/helix#10019] [General]
👋 while [upgrading rust to 1.77.0](https://github.com/Homebrew/homebrew-core/pull/166865), I have seen some regression build failure against 23.10 release, I believe this has been already fixed in master, but just need a new release. Thus creating this issue for it. Thanks!

**Comment by kirawi** [comment]
There will be a release soon: https://github.com/helix-editor/helix/pull/9946

**Reply by chenrui333** [reply]
cool, that is good to know, thanks for the pointer!
